processing/src/main/java/org/apache/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -319,129 +316,6 @@ protected IncrementalIndex(
     }
   }
 
-  public static class Builder
```

Comment:
so this static class has now changed to an abstract class with different concrete builder classes - nice